Assignment-22/countryStateDistrict/stateTriggerHelper

```diff
@@ -0,0 +1,83 @@
+public with sharing class stateTriggerHelper {
+     /*
+    ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+    * @Description      This method is used when an district or state is insert,update or delete.
+    * @author           Mohd Owais  <mohd.owais@fexle.com>
+    * @Return           void
+    * @Parameter        List,Map
+    * @created          2022-08-13
+    * ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+    */  
+    public static void insertUpdateDeleteState(List<State__c> newState,Map<Id,State__c> oldStateMap) {
```

Comment:
Is this method inserting, Updating state records? The name should be like rollUpStateAndDistrictCountOnCountry